docs/backup-restore.md

```diff
@@ -24,6 +24,34 @@ That means that you need to create a consistent snapshot by dumping all tables i
 For example, you can use `mysqldump` command with `--single-transaction` option in MySQL and `pg_dump` command in PostgreSQL to achieve that.
 Or when you use Amazon RDS (Relational Database Service) or Azure Database for MySQL/PostgreSQL, you can restore to any point within the backup retention period with the automated backup feature, which satisfies the requirement.
 
+## Cosmos DB
+
+You can create Cosmos DB account using `Continuous` backup policy to create continuous backups.
```

Comment:
You can but you don't need to?
If users need to enable `continuous backup`,  we should say so. 
If users don't need to do it, please explain what is alternative.
It sounds very blur.

Reply:
> You can but you don't need to?

Sorry, It is mandatory for continuous backup creation.

> If users need to enable continuous backup, we should say so.

Document updated with `You should create Cosmos DB account using Continuous backup policy to create continuous backups.`

> If users don't need to do it, please explain what is alternative.

The alternative method is not sufficient for transactionally consistent backup